src/common/forexApi.js

```diff
@@ -0,0 +1,6 @@
+// import axios from 'axios';
+
+// export default axios.create({
+//   baseUrl: 'https://freecurrencyapi.net/api/v2/historical',
+// });
+// // ?apikey=25c474a0-8926-11ec-8311-898ca4976dbd&base_currency=USD&date_from
```

Comment:
- [ ] Kindly remove this file as it is not being used. This will make your codebase cleaner and more professional